tenders-api/update_v7.sql

```diff
@@ -0,0 +1,18 @@
+
+/*
+Tenders DB Update script: v6 (buyer_user_details table)
+*/
+
+-- Drop table
+
+-- DROP TABLE public.buyer_user_details;
```

Comment:
Think these drop table comments can be removed - and a line added to the `drop_tables.sql` script. That should ensure it is always cleanly removed.

`DROP TABLE IF EXISTS buyer_user_details;`